templates/deno/README.md.twig

```diff
@@ -0,0 +1,30 @@
+# {{ spec.title }} {{sdk.name}} SDK
+
+![License](https://img.shields.io/github/license/{{ sdk.gitUserName|url_encode }}/{{ sdk.gitRepoName|url_encode }}.svg?v=1)
+![Version](https://img.shields.io/badge/api%20version-{{ spec.version|url_encode }}-blue.svg?v=1)
+{% if sdk.warning %}
+
+{{ sdk.warning }}
+{% endif %}
+
+{{ sdk.description }}
+
+
+
+![{{ spec.title }}]({{ sdk.logo }})
+
+## Installation
+
+To install via [NPM](https://www.npmjs.com/):
```

Comment:
Just read here that Deno isn't using any package manager:
https://github.com/denoland/deno/issues/47

From what I understand they are using import of remote git URLs, any idea how this should look like?

Reply:
Yea, I copied it from the TS templates and forgot to update it. later on add the repo through here https://deno.land/x